pkg/security/ebpf/probes/sysctl.go

```diff
@@ -0,0 +1,34 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux
+
+// Package probes holds probes related files
+package probes
+
+import manager "github.com/DataDog/ebpf-manager"
+
+func getSysCtlProbes(cgroup2MountPoint string) []*manager.Probe {
+	return []*manager.Probe{
+		{
+			ProbeIdentificationPair: manager.ProbeIdentificationPair{
+				UID:          SecurityAgentUID,
+				EBPFFuncName: GetSysCtlProbeFunctionName(),
+			},
+			CGroupPath: cgroup2MountPoint,
```

Comment:
I'm not familiar with how cgroup-based eBPF programs work. Is the eBPF program only called for processes that belong to the cgroup referenced by CGroupPath? If so, shouldn't we also care about other cgroups?

Reply:
The goal here is to point to the root cgroup, meaning it should include all processes. That said, you're right, I need to make sure we're _actually_ pointing to the root cgroup when the agent is containerized 👍🏻
I'll add something in my next commit, thanks !